src/data/lingui/en/messages.json

```diff
@@ -407,5 +406,7 @@
     "AssetsItem.About.title": "About",
     "AssetsItem.Transfers.title": "Transfers",
     "HistoryList.Item.title.cancel": "Cancel transfer",
-    "AssetsItem.About.Description.less": "Show less"
+    "AssetsItem.About.Description.less": "Show less",
+    "utils.address.getAddressName.address": "Address {index}",
```

Comment:
Context is needed only to tell translator, where is this string used. That's why we created "entities" namespace – it contains strings related to specific business-entity. Why did you change it to `utils`?